.github/workflows/ci.yml

```diff
@@ -26,8 +34,17 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
           CODECOV_STATIC_TOKEN: ${{ secrets.CODECOV_STATIC_TOKEN }}
-      - name: Run Tests and collect coverage
-        run: pytest --cov app ${{ env.CODECOV_ATS_TESTS }}
+      - name: Run Tests and collect coverage (if there are tests to run)
+        run: |
+          length_of_tests=$(cat codecov_ats/tests_to_run.json | jq 'length')
```

Comment:
Yeah, ideally can we put this echo in a different step of the workflow, and then another step that does just `pytest --cov app blah blah blah`. Basically if a user looks at this file, they should be able to copy+paste pretty easily

Reply:
the inability to do `pytest --cov app blah blah` is the reason the "blah blah" part was moved to a file in the first place.